airbyte-webapp/src/components/EntityTable/utils.tsx

```diff
@@ -98,6 +98,7 @@ export const getConnectionTableData = (
     lastSyncStatus: getConnectionSyncStatus(connection.status, connection.latestSyncJobStatus),
     connectorIcon: type === "destination" ? connection.source.icon : connection.destination.icon,
     entityIcon: type === "destination" ? connection.destination.icon : connection.source.icon,
+    connection,
```

Comment:
Is this pulling out and renaming a bunch of the properties from each ConnectionListItem, and then also returning a nested copy of the entire original ConnectionListItem object? 

Should we just pull out the additional property(/ies) that we want?

Reply:
Yeah so the issue is that we actually need (almost) the full `WebBackendConnectionListItem` (or `WebBackendConnectionRead`) to call `syncConnection(connection)` in `StatusCellControl.tsx`. I would rather not pull out and drill through the additional 6 properties, instead just pass through `connection`.

Agreed that it's weird that we are pulling out and renaming a bunch of properties here. I was also surprised to come across that. Probably the better option would be to just pass through the `connection` object and do all this renaming closer to the consumer. I did not want to go into that refactoring in this PR though!